main.py

```diff
@@ -2,40 +2,36 @@
 import glob
 import re
 import os
-# Ścieżka do folderu, w którym znajdują się pliki
-folder_path = os.getcwd() + '\data'
-
+# Path to the folder where the files are located
+folder_path = os.path.join(os.getcwd(), 'data')
+file_extension = ['xlsx', 'csv', 'txt']
+files = []

```

Comment:
This should not be global variable, it's not a predefine setting and can couse errors when used in multiple functions